### PR TITLE
[WIP] Fix thing creation for updated XML

### PR DESCRIPTION
**Pull request by J-N-K**
was: https://github.com/eclipse/smarthome/pull/6537

Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by martinvw** [comment]
@J-N-K the build seems unhappy

**Comment by J-N-K** [comment]
That‘s expected. I just created the PR because I would forget otherwise :)

File: bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ManagedThingProvider.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2014,2019 Contributors to the Eclipse Foundation
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
```
Comment (wborn) [inline bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ManagedThingProvider.java]: Please use the openHAB license header instead.

**Comment by J-N-K** [comment]
We should continue with #628

**Pull request closed without merging**